specs/Deployment/DeploymentAPI.md

```diff
@@ -234,6 +251,7 @@ Here's the previous example extended with the `Show UI on error` option:
         // These should be run on a separate thread so as not to hang your app while the
         // packages deploy.
         var initializeTask = Task.Run(() => DeploymentManager.Initialize(deploymentInitializeOptions));
+        //...do other work while the repair is running...
```

Comment:
Again, the comment says "repair", but the code says "Initialize". Please get them to agree on which is correct.